test/code-build-test.js

```diff
@@ -173,6 +179,9 @@ describe("inputs2Parameters", () => {
     expect(test)
       .to.haveOwnProperty("buildspecOverride")
       .and.to.equal(undefined);
+    expect(test).to.haveOwnProperty("computeTypeOverride").and.to.equal(undefined);
+    expect(test).to.haveOwnProperty("environmentTypeOverride").and.to.equal(undefined);
+    expect(test).to.haveOwnProperty("imageOverride").and.to.equal(undefined);
```

Comment:
Can we have a test case where the overrides are specified?

Reply:
Resolved